templates/zammad/config.yml

```diff
@@ -1,7 +1,8 @@
 name: Zammad
 description: |
   Zammad is a web based open source helpdesk/customer support system
-version: "1.2"
+version: "2.2.0-12"
+defaultVersion: "2.2.0-12"
```

Comment:
`defaultVersion`parameter doesn't exist. Please remove it.
`version` parameter at this file, defines the default version.
```
-defaultVersion: "2.2.0-12"
```